### PR TITLE
[pyAMQP] Align pyAMQP

### PR DESCRIPTION
**Pull request by l0lawrence**

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py [hunk 1]
```diff
@@ -737,11 +742,12 @@ def connect(self):
             ) from None
         try:
             self.sock = create_connection(
-                url="wss://{}".format(self._custom_endpoint or self._host),
+                url="wss://{}".format(self._custom_endpoint or self._host) if self._use_tls
+                    else "ws://{}".format(self._custom_endpoint or self._host),
```
Comment (swathipil) [inline sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py:746]: no changes need to be made in the _pyamqp_transport file to support this? or expose the option to not use tls in public api?

**Review by swathipil — APPROVED** [review]

**Review by kashifkhan — APPROVED** [review]

File: sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py (hunk 1 above)
Comment (l0lawrence) [reply sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py:746]: I don't believe we have an emulator for SB at this time so use_tls wouldn't be used, I don't believe it is a part of public api regardless -- it is taken from the conn_str parsing in EH

Comment (swathipil) [reply sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py:746]: sounds good, we'll prob end up adding stuff when we need to for features anyway

**Pull request merged**